README.md

```diff
@@ -18,13 +18,17 @@ Telephony.config do |c|
   c.twilio.timeout = 5 # This is optional. The default is `5`
   c.twilio.record_voice = false # This is optional. The default is `false`
 
-  c.pinpoint.sms.region = 'us-west-2' # This is optional, us-west-2 is the default
-  c.pinpoint.sms.application_id = 'fake-pinpoint-application-id-sms'
-  c.pinpoint.sms.shortcode = '123456'
-  c.pinpoint.sms.longcode_pool = ['+12223334444', '+15556667777']
+  c.pinpoint.add_sms_config do |sms|
+    sms.region = 'us-west-2' # This is optional, us-west-2 is the default
+    sms.application_id = 'fake-pinpoint-application-id-sms'
+    sms.shortcode = '123456'
+    sms.longcode_pool = ['+12223334444', '+15556667777']
```

Comment:
Does is make sense to test w/ an SMS longcode pool. I think we got rid of that entirely when we retired the SMS longcode pool adapter. Doesn't look like it comes back into use anywhere in this PR.

Reply:
Ok I'll remove it